test/integration/transaction.test.js

```diff
@@ -142,6 +142,16 @@ describe(Support.getTestDialectTeaser('Transaction'), function() {
     ).to.eventually.be.rejected;
   });
 
+  it('does not allow queries immediatly after rollback call', function() {
+    var self = this;
+    return expect(
+      this.sequelize.transaction().then(function(t) {
+        t.rollback();
+        return self.sequelize.query('SELECT 1+1', {transaction: t, raw: true});
```

Comment:
It's failing for sqlite because the rollback query is still running after the ```SELECT 1+1``` query call, and it expects no running queries when the test finishes. I can't think of any ideas to prevent this and at the same time return the query promise that will be rejected.